get-started/deployment-options.md

```diff
@@ -1,17 +1,21 @@
 ---
+navigation_title: Deployment options
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro-deploy.html
 products:
   - id: elasticsearch
 ---
 
-# Deployment options [elasticsearch-intro-deploy]
+# Deployment options
 
-Use this page for a quick overview of your options for deploying Elastic.
+You can run Elastic on virtually any infrastructure, allowing you to choose the model that best fits your operational needs. 
 
-:::{include} /deploy-manage/_snippets/deployment-options-overview.md
-:::
+These are the most common deployment types and their main features:
```

Comment:
would be helpful to explain why this decision is impactful: each of these deployment types changes the amount of stuff that can be done across deployments and the amount of manual work you need to put in to stand up an environment

we kind of talk about it in the [orchestration](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2795/deploy-manage/deploy#about-orchestration) explanation

Reply:
Added a section right before.